libs/ledger-live-common/src/families/aptos/LedgerAccount.ts

```diff
@@ -48,49 +52,50 @@ export default class LedgerAccount {
     return this.hdPath;
   }
 
-  address(): HexString {
+  address(): AccountAddress {
     return this.accountAddress;
   }
 
-  authKey(): HexString {
+  authKey(): AccountAddress {
     const hash = sha3Hash.create();
-    hash.update(this.publicKey);
+    hash.update(this.publicKey.toString("hex"));
     hash.update("\x00");
-    return HexString.fromBuffer(hash.digest());
+    return AccountAddress.from(hash.digest());
   }
 
-  pubKey(): HexString {
-    return HexString.fromBuffer(this.publicKey);
+  pubKey(): AccountAddress {
+    return AccountAddress.from(this.publicKey.toString("hex"));
   }
 
-  async asyncSignBuffer(buffer: Uint8Array): Promise<HexString> {
+  async asyncSignBuffer(buffer: Uint8Array): Promise<Hex> {
     if (!this.client) {
       throw new Error("LedgerAccount not initialized");
     }
     const response = await this.client.signTransaction(this.hdPath, Buffer.from(buffer));
-    return HexString.fromBuffer(response.signature);
+    return Hex.fromHexString(response.signature.toString("hex"));
```

Comment:
Does it make sense to somehow use that `signature` is Buffer that extends Uint8Array and Hex is Uint8Array?